build/configs/qemu/tash_16m/Make.defs

```diff
@@ -137,5 +137,5 @@ HOSTCFLAGS = -Wall -Wstrict-prototypes -Wshadow -g -pipe
 HOSTLDFLAGS =
 
 define DOWNLOAD
-	@gnome-terminal -e "qemu-system-arm -M lm3s6965evb -kernel $(TOPDIR)/../build/output/bin/tinyara -nographic"
+	@gnome-terminal -e "qemu-system-arm -M lm3s6965evb -kernel $(TOPDIR)/../build/output/bin/tinyara -nographic -gdb tcp::3333"
```

Comment:
DOWNLOAD and "running tinyara on qemu doesn't match"
How about introducing "make run" ?